index.html

```diff
@@ -13,25 +13,144 @@
-                <a href="https://github.com/mohametalmeari"><i class="fa fa-github"></i></a>
-                <a href=""><i class="fa fa-instagram"></i></a>
-            </div>
+            <ul class="icons">
+                <li><a href="https://twitter.com/MohametAlmeari"><i class="fa fa-twitter"></i></a></li>
+                <li><a href="https://www.linkedin.com/in/mohamet-almeari"><i class="fa fa-linkedin"></i></a></li>
+                <li><a href="https://github.com/mohametalmeari"><i class="fa fa-github"></i></a></li>
+                <li><a href=""><i class="fa fa-instagram"></i></a></li>
+            </ul>
         </section>
+        
+            <section class="works">
+                <div class="card">
+                    <div class="preview pv1">
+                    <img src="/images/website_example.jpg" alt="Website Preview">
+                    </div>
+                    <h3 >
+                        Sky Design
+                    </h3>
+                    <div class="type_and_date">
+                        <span>CANOPY</span>
+                        <span>●</span>
+                        <span>Front End Dev</span>
+                        <span>●</span>
+                        <span>2023</span>
+                    </div>
+                    <p>
+                        Interior and exterior building desgin, get your dream house decoration.
+                    </p>
+                    <ul class="prog_lang">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button type="button">
+                        See Project
+                    </button>
+                </div>
+                <div class="card">
+                    <div class="preview pv2">
+                    <img src="/images/website_example.jpg" alt="Website Preview">
+                    </div>
+                    <h3 >
+                        Sky Design
+                    </h3>
+                    <div class="type_and_date">
+                        <span>CANOPY</span>
+                        <span>●</span>
+                        <span>Front End Dev</span>
+                        <span>●</span>
+                        <span>2023</span>
+                    </div>
+                    <p>
+                        Interior and exterior building desgin, get your dream house decoration.
+                    </p>
+                    <ul class="prog_lang">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button type="button">
+                        See Project
+                    </button>
+                </div>
+                <div class="card">
+                    <div class="preview pv3">
+                    <img src="/images/website_example.jpg" alt="Website Preview">
```

Comment:
- [ ] As I mentioned earlier it's better if you try to stay consistent with the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0-1&t=sBtzHwNqjzdJ4wpR-0) as much as you can at least until you start building your own projects then you can replace the project images with your own.
    - Make sure to export and add the project images shown in the design ✔

Reply:
Hi @AbelG101, I did all changes, could you check again if there is any other mistake before I submit.